README.rst

```diff
@@ -66,8 +66,8 @@ Execute via the Python API
    import papermill as pm
 
    pm.execute_notebook(
-      notebook_path = 'path/to/input.ipynb',
-      output_path = 'path/to/output.ipynb',
+      src = 'path/to/input.ipynb',
+      dst = 'path/to/output.ipynb',
```

Comment:
I love this naming. For backwards compatibility though, we would need to make `notebook_path` and `output_path` be deprecated and pass it through on the new `src` and `dst` keyword args.

Reply:
I just noticed that this needs to go one of two ways here, because no changes were made to `execute_notebook`'s function:

Either we'll need to change `execute_notebook` or this example should go without named arguments (for now).

Reply:
I think for this one that unnamed arguments are the way to go esp since the files are name input and output. Agree on the deprecation and pass through. (Wasn't sure how many folks were using.)

Reply:
At least for my friendly Netflix colleagues, it's in the dozens. We can always make this a short deprecation cycle, I just like to not cause too much thrashing for people. I did talk to Eric about making the papermill interface allow publishing too, which would also change this fundamentally too.

Reply:
Let's do this. I'm going to leave the interface as is for now. When we handle publishing, we can extend/deprecate/etc. the interface. I changed the naming before I updated the formatting and the format change got rid of most of the long name yuckiness.